src/extended.ts

```diff
@@ -30,6 +31,7 @@ interface FullWebAuthnExtensionsJSON {
 
 const authenticationExtensionsClientInputsSchema: Schema = {
   appid: optional(copyValue),
+  appidExclude: optional(copyValue),
```

Comment:
I only added the extension identifier to the extensions input schema given that, [according to the spec](https://w3c.github.io/webauthn/#sctn-appid-exclude-extension), this extension doesn't return any outputs.

Reply:
Well, seems that this is changing soon... https://github.com/w3c/webauthn/issues/1445 🙃

Reply:
I think this should also go in the main def. Would you mind updating this PR to include it in `webauthn-schema.ts`?

Reply:
Sure! Anyway, please refer to the [other issue that I created](https://github.com/github/webauthn-json/issues/23) because even if we declare the extension in the main def, we are not getting the `clientExtensionResults` in the `get` function from the main def. 🙂